### ISSUE TITLE
Tables: Change old chart for new design

### ISSUE THREAD

**Issue by Nico-Kestra** [kestra-io/kestra#6175]
### Describe the issue

In the tables, the new design of the charts should be applied on executions statistics row, using the Kestra color scheme (purple) or classic.

<img width="515" alt="Capture d’écran 2024-11-28 à 16 35 53" src="https://github.com/user-attachments/assets/13bd2579-e2f7-4b98-911f-5d638899432d">

Design expected:

<img width="614" alt="Capture d’écran 2024-11-28 à 16 39 31" src="https://github.com/user-attachments/assets/2fc59a94-3dcd-458b-857c-01e59ed0f5f9">

<img width="757" alt="Capture d’écran 2024-11-28 à 16 39 48" src="https://github.com/user-attachments/assets/1ee289ea-c40f-4926-95a5-1f7ffe124d6b">

### Environment

- Kestra Version: develop

**Comment by JoeCelaster** [comment]
I can work on this issue, Please assign this to me

**Comment by MilosPaunovic** [comment]
I've just assigned another issue to you @JoeCelaster, why don't we close that one first and then tackle this.

**Comment by JoeCelaster** [comment]
@MilosPaunovic Okay Fine .

**Comment by SaumyaG1318** [comment]
I want to work on this issue can this be assigned to me.

**Comment by MilosPaunovic** [comment]
Absolutely, go for it @SaumyaG1318! 🚀

**Comment by MilosPaunovic** [comment]
Is there any progress on this @SaumyaG1318?

**Comment by SaumyaG1318** [comment]
> Is there any progress on this @SaumyaG1318?

Yes I have identified the component that I have to modify in order to change the chart designs and I think I can get it done in a day or two.

**Comment by SaumyaG1318** [comment]
![image](https://github.com/user-attachments/assets/3e6ce11b-031b-4899-964e-a746477859d3)

Is this the intended design?
![image](https://github.com/user-attachments/assets/eb5cc569-59ef-40f1-865c-fb00abd62a2b)
I couldn't figure out how to remove these hills like borders so I just make the border width 0 and gave it any color and made it transparent like this.
```
//In line 132 in State Chart.vue remove line
borderRadius: 4,
//Remove const darkTheme from line 122 before applying these in StateChart.vue around line 150
borderWidth: 0,
backgroundColor: Utils.hexToRgba("#ffffff", 0.00001),
```
@MilosPaunovic Is it possible to directly remove this hill like border without working around it by making it transparent.

**Comment by SaumyaG1318** [comment]
I this this is is probably the right way to make the background transparent because the background is going to render with the component anyhow. So @MilosPaunovic can you review the design, and verify were these the expected changes if so I will put up a pull request.

**Comment by MilosPaunovic** [comment]
Hi @SaumyaG1318,

The work you've done on this was not exactly what we were looking to achieve.

I've marked the places where we need to do the change (as part of your PR), and we want the component to be replaced with the matching one from the `kestra/ui/src/components/dashboard/components/charts` folder.

I'll unassign you for now. However, if you do open a PR, it will be reviewed, and you'll be reassigned if no one else handles it in the meantime.

**Comment by Ben8t** [comment]
@SaumyaG1318 did you get a chance to review Milos feedback?

**Comment by Tal-Assaf05** [comment]
My name is Tal I am a Computer Science student at FIU currently enrolled in the "Internship Ready Software Development" course, and as part of the class, my team and I are interested in contributing to enhancing and improving your project.

I personally would be thrilled to take on this specific task and would be happy to get started right away. If you could kindly provide access to the project, I am eager to begin working on it.

Thank you for considering this request. I look forward to the opportunity to contribute to the project.

**Comment by MilosPaunovic** [comment]
Hi @Tal-Assaf05,

This issue is already assigned, but may I suggest that you check our [curated list of good first issues](https://go.kestra.io/contributing) to see if there is something you find interesting enough and think you can tackle independently. It can be further filtered by desired labels to narrow down the results (suggesting to filter by either `area/backend` or `area/frontend` ones, based on your preferences).
